src/sentry/conf/server.py

```diff
@@ -2509,6 +2513,90 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     (("email", "Read email address and verification status. Requires openid scope."),),
 )
 
+"""
+    This list is tracking old api endpoints that don't correctly implement pagination.
+    The goal is to eventually add pagination for all and shrink this list.
+    DO NOT ADD ANY NEW APIS
+"""
+SENTRY_API_PAGINATION_ALLOWLIST = {
```

Comment:
Can you please define this list in its own file in sentry/conf and here say:
```
SENTRY_API_PAGINATION_ALLOWLIST = SENTRY_API_PAGINATION_ALLOW_LIST_DO_NOT_MODIFY
```
The reason is this way we can add ourselves as reviewers of that file and make sure we get to review every change on it.